### DISCUSSION TITLE
How do the 3 different speeds work? What is disregarded in each speed setting?

### DISCUSSION THREAD

**Discussion by Teedo145** [lllyasviel/Fooocus#1562] [Q&A]
For example, the negative prompt is disregarded in Extreme Speed...what else? and what disregarded in Speed?

**Comment by eddyizm** [comment]
This is a really good question. Whatever the answer is should be incorporated into the read me.

**Comment by mashb1t** [comment]
- Quality => 60 steps, sampler etc. what you selected in developer tools
see https://github.com/lllyasviel/Fooocus/blob/main/modules/async_worker.py#L177
- Speed => 30 steps, sampler etc. what you selected in developer tools
see https://github.com/lllyasviel/Fooocus/blob/main/modules/async_worker.py#L174
- Extreme Speed => 8 steps, LCM LoRA, various options disabled (negative prompt, refiner, cfg scale, sharpness, sampler, etc.)
see https://github.com/lllyasviel/Fooocus/blob/main/modules/async_worker.py#L187-L197

This is also why i decided to add additional labels like in the aspect ratios to my fork, see https://github.com/mashb1t/Fooocus/blob/main/modules/flags.py#L37-L41

![image](https://github.com/lllyasviel/Fooocus/assets/9307310/9555e0ef-455a-4dbf-bf5d-8e55357e92bc)
